app/routes.py

```diff
@@ -246,6 +255,20 @@ def add_comment():
 
 # Get all comments from db
 @app.route("/comments", methods=["GET"])
+@jwt_required()
 def manage_comments():
     comments = Comment.get_all()
     return jsonify(Comment.__schema__(many=True).dump(comments)), 200
+
+
+# Authentication method for username and password
+@app.route("/login", methods=["POST"])
+@jwt_required()
```

Comment:
The /login is always no `jwt_required`. The user wants to get the jwt token somewhere in order to use it to other endpoints.